typescript/token/src/globalTypes.ts

```diff
@@ -0,0 +1,3 @@
+export type DomainId = number;
+export type Caip2Id = `${string}:${string}`; // e.g. ethereum:1 or solana:mainnet-beta
```

Comment:
I recently renamed these in the warp UI to ChainCaip2Id and TokenCaip19Id for clarity. let's please do the same here.